fsharp.extraction.targets

```diff
@@ -14,7 +20,7 @@
                 3. fsti files are filtered out
                 4. the order of files is unchanged.
     -->
-    <UsingTask TaskName="TransformFStarSourceFiles" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll" >
+    <UsingTask TaskName="TransformFStarSourceFiles" TaskFactory="$(CodeTaskFactory)" AssemblyFile="$(CodeTaskAssemblyFile)" >
```

Comment:
Parameterizing the CodeTaskFactory allows to consume ulibfs.dll from both framework and sdk-style projects.